apps/docs/components/Navigation/NavigationMenu/GlobalNavigationMenu.tsx

```diff
@@ -0,0 +1,198 @@
+                    <div className="p-3 md:p-1">
+                      {section[0].menuItems?.map((menuItem, menuItemIndex) => (
+                        <Fragment key={`desktop-docs-menu-section-${menuItemIndex}`}>
+                          {menuItemIndex !== 0 && <MenubarSeparator className="bg-border-muted" />}
+                          {menuItem.map((item) =>
+                            !item.href ? (
+                              <div className="font-mono tracking-wider flex items-center text-foreground-muted text-xs uppercase rounded-md p-2 leading-none">
+                                {item.label}
+                              </div>
+                            ) : (
+                              <NavigationMenuLink asChild>
+                                <MenuItem
+                                  href={item.href}
+                                  title={item.label}
+                                  community={item.community}
+                                  icon={item.icon}
+                                />
+                              </NavigationMenuLink>
+                            )
+                          )}
+                        </Fragment>
+                      ))}
+                    </div>
+                  </NavigationMenuContent>
+                </NavigationMenuItem>
+              ) : (
+                <NavigationMenuItem
+                  key={`desktop-docs-menu-section-${section[0].label}`}
+                  className="text-sm relative h-full"
+                >
+                  <NavigationMenuLink asChild>
+                    <Link
+                      href={section[0].href}
+                      className={cn(
+                        navigationMenuTriggerStyle(),
+                        triggerClassName,
+                        activeLabel === section[0].label && 'text-foreground border-foreground'
+                      )}
+                    >
+                      {section[0].label === 'Home' ? (
+                        <HomeMenuIconPicker icon={section[0].icon} />
+                      ) : (
+                        section[0].label
+                      )}
+                    </Link>
+                  </NavigationMenuLink>
+                </NavigationMenuItem>
+              )
+            )}
+          </NavigationMenuList>
+        </ResponsiveScrollArea>
+      </NavigationMenu>
+    </div>
+  )
+}
+
+const ResponsiveScrollArea: FC<PropsWithChildren<{ isLowerThanMd: boolean }>> = ({
+  isLowerThanMd,
+  children,
+}) =>
+  !isLowerThanMd ? (
+    <>{children}</>
+  ) : (
+    <>
+      <ScrollArea className="relative">
+        <div className="absolute h-full right-0 w-7 z-50 bg-gradient-to-r from-transparent to-background" />
```

Comment:
I _think_ NavigationMenu is already on z-50, but just be careful using z indexes with 50 and above.